### ISSUE TITLE
How  to compile hidapi  library for android use hidraw backend

### ISSUE THREAD

**Issue by chenzhaoting** [libusb/hidapi#648]
in android ,there is no libudev.h or libudev.so

**Comment by chenzhaoting** [comment]
> hidraw backend is not advertized to be able to compile on Android, and for libusb backend you don't need udev

yes,if i compiled use libusb all is ok ,but my device is HID over I2C ,hidraw backend  should be used

**Comment by Youw** [comment]
Yes it would be nice if that would be possible.
There is no support for it now.
